analyticsclient/users.py

```diff
@@ -0,0 +1,29 @@
+import analyticsclient.constants.data_format as DF
+
+
+class UserProblemWeeklyData(object):
+    """ Get user problem history per week. """
+
+    def __init__(self, client, course_id, user_id):
+        """
+        Initialize the API client.
+
+        Arguments:
+
+            client (analyticsclient.client.Client): The client to use to access remote resources.
+            user_id (int): The ID of the user
+
+        """
+        self.client = client
+        self.course_id = course_id
+        self.user_id = unicode(user_id)
```

Comment:
Nit: I think this should only be converted to `unicode` at the time you create the URL.